lib/app.rb

```diff
@@ -0,0 +1,32 @@
+
+require 'sinatra/base'
+#require 'byebug'
+# require 'pry'
```

Comment:
Det är naturligtvis alldeles rätt, ska försöka bättra mig på det. Jag har väl dragit gränsen vad som kan copy/pastas litet för slö (all web kopierar jag rakt av, det mesta av allt som bara är en uppräkning av "requires" också, men det är kanske litet för brett....däremot all logik-ruby försöker jag skriva in för hand men require och pekningar bör också in i handskrivandet.